docs/gtfs/Gtfs_Instance_class_diagram.md

```diff
@@ -0,0 +1,25 @@
+# GtfsInstance
+
+<header>
```

Comment:
Header appears as raw html code in readme preview.

Reply:
Removed header as styling not important and it seems GitHub markdown doesn't like to play ball with CSS sheets.